README.md

```diff
@@ -116,23 +143,29 @@ This repository uses [Prisma](https://www.prisma.io/) as an ORM for interacting
 
 ### Running and inspecting the database locally
 
-1. Fill in `PG_*` variables in `.env` using samples from `.env.example` as a guide
-2. Spin up dependencies using `npm run start:sandbox`
+1. Ensure `PG_*` variables in `.env` are filled in. Values from `.env.example` should work
+2. Spin up the database using `npm run start:sandbox`
 3. Using IntelliJ or whatever tool you use for inspecting databases, add a database using fields from from `.env`
 
+> If you aren't seeing the tables in IntelliJ, you may have to select the right schemas from the 'Schemas' tab in the Data Sources window.
+
 ### Running migrations
 
 To run a database migration do the following:
 
 1. Make any schema additions in `prisma/schema.prisma`
 2. Spin up dependencies using `npm run start:sandbox`. This will also run the `prisma migrate dev` command, applying all
-   existing migrations. See [the docs](https://www.prisma.io/docs/concepts/components/prisma-migrate/migrate-development-production#development-environments)
+   existing migrations. See the [Prisma docs](https://www.prisma.io/docs/concepts/components/prisma-migrate/migrate-development-production#development-environments)
    for more info
 3. To create a new migration after making schema changes, with the sandbox already running, run `npm run db:migrate --name <migration_name>` - this will create your migration in a new folder under `prisma/migrations`. If you omit the `--name` option, you will be prompted to name the migration
 4. Running `prisma migrate dev` will trigger generation of artifacts automatically, but you can trigger these manually
    by running `npm run db:generate` to rebuild the `@prisma/client`, which provides type safety and utility functions
    for any newly added tables and fields
 
+## Logging
+
+The app uses [pino](https://github.com/pinojs/pino) and [pino-http](https://github.com/pinojs/pino-http) for logging. The logger is configured in `/src/infrastructure/logger.ts`.
```

Comment:
Maybe also point to the request logger middleware in `src/web/middleware/http-logger-middleware.ts`?